### DISCUSSION TITLE
A Telegram group

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#360] [Ideas]
Most people(like me) tend to spend most of the time on telegram.
I could notice that Flet has no telegram group.
Creating this could be very useful and great. (Ex for announcements, discussions etc)

What do you think ?

**Comment by PaladinXIV** [comment]
Let me know if you find one.

**Reply by a1y9** [reply]
https://t.me/python_flet

**Reply by PaladinXIV** [reply]
I been super busy and just say this. 
Thank You so much!!
